### DISCUSSION TITLE
How to remove extra spaces made when view being scrolled to the input

### DISCUSSION THREAD

**Discussion by bryanprimus** [mateusz1913/react-native-avoid-softinput#176] [Q&A]
### Current behavior (Both ios and android)

https://github.com/mateusz1913/react-native-avoid-softinput/assets/46083126/0493a6fe-4cf2-4b03-a92b-391b09123d44

There is an extra spaces made below the container after the view scrolled to the input

I'm not sure this is intentional or not. but i'm currently finding a way to remove it.

Also, when the view is scrolling the input, is it possible to configure it so that the view scrolls exactly to where the bottom of the input position is? I believe the library adds some offset or spaces so that it scrolls slightly below the active input 

### Expected behavior:

I use `keyboardavoidingview` for this with behavior `height` but this only works on ios.

https://github.com/mateusz1913/react-native-avoid-softinput/assets/46083126/de866141-f024-4599-b50c-f264cc1e1328

**Comment by mateusz1913** [comment]
👋  @bryanltobing, the answer to your first question is the bottom safe inset - it's intentional, as there's no sense from the UX perspective to put any UI element that is interactive (e.g. button or input) below the bottom safe inset.

Regarding the second question, it's hard to say, there're many factors that can play their role when computing that scroll offset. From the UI/DOM structure of your form, to the position of the scrollview.
What you can do is try to use [AvoidSoftInput.setAvoidOffset](https://mateusz1913.github.io/react-native-avoid-softinput/docs/api/module/set-avoid-offset/) function and see if it will work in your case
